apps/ledger-live-mobile/src/components/DBSave.ts

```diff
@@ -31,13 +34,13 @@ export default function useDBSaveEffect<D, S>({
       // nb it does not prevent race condition here. save must be idempotent and atomic
       throttleFn(async () => {
         const { lense, save, state, getChangesStats } = latestProps.current;
-        const changedStats = getChangesStats(lastSavedState.current, state); // we compare last saved with latest state
-
-        if (!changedStats) return; // if it's falsy, it means there is no changes
-
-        await save(lense(state), changedStats); // we save it for real
+        if (lastSavedState?.current && state) {
+          const changedStats = getChangesStats(lastSavedState.current, state); // we compare last saved with latest state
```

Comment:
Wouldn't this be getChange**d**Stats, just a comment, it's been like that forever

Reply:
probabyl XD